File: BACKEND/prontoMiao/src/main/java/it/unical/prontoMiao/request/AuthRequest.java
package it.unical.prontoMiao.request;

public class AuthRequest {
    private String email;
    private String password;
    private String tipoUtente;
    private String nome;
    private String descrizione;
    private String orari;
    private String eventi;
    private String indirizzo;
    private String cognome;
    private String cf;
    private String telefono;
    private String preferenze;
    private String condizioni_abitative;
    private String informazioni_aggiuntive;

    public AuthRequest() {
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    public String getTipoUtente() {
        return tipoUtente;
    }

    public void setTipoUtente(String tipoUtente) {
        this.tipoUtente = tipoUtente;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public String getDescrizione() {
        return descrizione;
    }

    public void setDescrizione(String descrizione) {
        this.descrizione = descrizione;
    }

    public String getOrari() {
        return orari;
    }

    public void setOrari(String orari) {
        this.orari = orari;
    }

    public String getEventi() {
        return eventi;
    }

    public void setEventi(String eventi) {
        this.eventi = eventi;
    }

    public String getIndirizzo() {
        return indirizzo;
    }

    public void setIndirizzo(String indirizzo) {
        this.indirizzo = indirizzo;
    }

    public String getCognome() {
        return cognome;
    }

    public void setCognome(String cognome) {
        this.cognome = cognome;
    }

    public String getCf() {
        return cf;
    }

    public void setCf(String cf) {
        this.cf = cf;
    }

    public String getTelefono() {
        return telefono;
    }

    public void setTelefono(String telefono) {
        this.telefono = telefono;
    }

    public String getPreferenze() {
        return preferenze;
    }

    public void setPreferenze(String preferenze) {
        this.preferenze = preferenze;
    }

    public String getCondizioni_abitative() {
        return condizioni_abitative;
    }

    public void setCondizioni_abitative(String condizioni_abitative) {
        this.condizioni_abitative = condizioni_abitative;
    }

    public String getInformazioni_aggiuntive() {
        return informazioni_aggiuntive;
    }

    public void setInformazioni_aggiuntive(String informazioni_aggiuntive) {
        this.informazioni_aggiuntive = informazioni_aggiuntive;
    }
}

File: FRONTEND/Centro/src/app/model/AuthRequest.ts
class AuthRequest {
    private email: string;
    private password: string;
    private tipoUtente: string;
    private nome: string;
    private descrizione: string;
    private orari: string;
    private eventi: string;
    private indirizzo: string;
    private cognome: string;
    private cf: string;
    private telefono: string;
    private preferenze: string;
    private condizioni_abitative: string;
    private informazioni_aggiuntive: string;
  
    constructor() {}
  
    getEmail(): string {
      return this.email;
    }
  
    setEmail(email: string): void {
      this.email = email;
    }
  
    getPassword(): string {
      return this.password;
    }
  
    setPassword(password: string): void {
      this.password = password;
    }
  
    getTipoUtente(): string {
      return this.tipoUtente;
    }
  
    setTipoUtente(tipoUtente: string): void {
      this.tipoUtente = tipoUtente;
    }
  
    getNome(): string {
      return this.nome;
    }
  
    setNome(nome: string): void {
      this.nome = nome;
    }
  
    getDescrizione(): string {
      return this.descrizione;
    }
  
    setDescrizione(descrizione: string): void {
      this.descrizione = descrizione;
    }
  
    getOrari(): string {
      return this.orari;
    }
  
    setOrari(orari: string): void {
      this.orari = orari;
    }
  
    getEventi(): string {
      return this.eventi;
    }
  
    setEventi(eventi: string): void {
      this.eventi = eventi;
    }
  
    getIndirizzo(): string {
      return this.indirizzo;
    }
  
    setIndirizzo(indirizzo: string): void {
      this.indirizzo = indirizzo;
    }
  
    getCognome(): string {
      return this.cognome;
    }
  
    setCognome(cognome: string): void {
      this.cognome = cognome;
    }
  
    getCf(): string {
      return this.cf;
    }
  
    setCf(cf: string): void {
      this.cf = cf;
    }
  
    getTelefono(): string {
      return this.telefono;
    }
  
    setTelefono(telefono: string): void {
      this.telefono = telefono;
    }
  
    getPreferenze(): string {
      return this.preferenze;
    }
  
    setPreferenze(preferenze: string): void {
      this.preferenze = preferenze;
    }
  
    getCondizioniAbitative(): string {
      return this.condizioni_abitative;
    }
  
    setCondizioniAbitative(condizioni_abitative: string): void {
      this.condizioni_abitative = condizioni_abitative;
    }
  
    getInformazioniAggiuntive(): string {
      return this.informazioni_aggiuntive;
    }
  
    setInformazioniAggiuntive(informazioni_aggiuntive: string): void {
      this.informazioni_aggiuntive = informazioni_aggiuntive;
    }
  }
  